packages/nextjs/pages/batches/index.tsx

```diff
@@ -0,0 +1,271 @@
+import Image from "next/image";
+import Link from "next/link";
+import { Footer } from "~~/components/Footer";
+import { MetaHeader } from "~~/components/MetaHeader";
+import TrackedLink from "~~/components/TrackedLink";
+
+const NEXT_BATCH_NUMBER = 10;
+
+const BATCH_CARDS_INFO = [
+  {
+    name: "Batch #1",
+    participants: 13,
+    startDate: "Jan 13",
+    batchPageLink: "https://batch1.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch1.buidlguidl.com",
+  },
+  {
+    name: "Batch #2",
+    participants: 4,
+    startDate: "Feb 04",
+    batchPageLink: "https://batch2.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch2.buidlguidl.com",
+  },
+  {
+    name: "Batch #3",
+    participants: 3,
+    startDate: "Feb 29",
+    batchPageLink: "https://batch3.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch3.buidlguidl.com",
+  },
+  {
+    name: "Batch #4",
+    participants: 18,
+    startDate: "Apr 03",
+    batchPageLink: "https://batch4.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch4.buidlguidl.com",
+  },
+  {
+    name: "Batch #5",
+    participants: 8,
+    startDate: "Apr 26",
+    batchPageLink: "https://batch5.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch5.buidlguidl.com",
+  },
+  {
+    name: "Batch #6",
+    participants: 9,
+    startDate: "Jun 02",
+    batchPageLink: "https://batch6.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch6.buidlguidl.com",
+  },
+  {
+    name: "Batch #7",
+    participants: 10,
+    startDate: "Jul 01",
+    batchPageLink: "https://batch7.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch7.buidlguidl.com",
+  },
+  {
+    name: "Batch #8",
+    participants: 10,
+    startDate: "Jul 27",
+    batchPageLink: "https://batch8.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch8.buidlguidl.com",
+  },
+  {
+    name: "Batch #9",
+    participants: 11,
+    startDate: "Sep 14",
+    batchPageLink: "https://batch9.buidlguidl.com/",
+    githubRepoLink: "https://github.com/BuidlGuidl/batch9.buidlguidl.com",
+  },
+];
+
```

Comment:
Note: @phipsae is working on getting this data into the BG firebase database, so soon we'd have an API endpoint to get all the batch data.

Reply:
Creating an issue to change the hardcoded data to the API endpoint whenever is ready 🙏